dll/win32/devmgr/lang/ro-RO.rc

```diff
@@ -19,17 +20,17 @@ BEGIN
     IDS_ENABLEDEVICE "Folosește acest dispozitiv (activare)"
     IDS_DISABLEDEVICE "Nu folosi acest dispozitiv (dezactivare)"
     IDS_UNKNOWNDEVICE "Dispozitiv necunoscut"
-    IDS_NODRIVERLOADED "Pentru acest dispozitiv nu sunt instalate modúle-pilot."
+    IDS_NODRIVERLOADED "Pentru acest dispozitiv nu sunt instalate module-pilot."
     IDS_DEVONPARENT "pe %1"
     IDS_TROUBLESHOOTDEV "&Depanare…"
     IDS_ENABLEDEV "A&ctivare"
     IDS_REINSTALLDRV "Reinstalea&ză modulul-pilot"
     IDS_PROPERTIES "P&roprietăți"
-    IDS_UPDATEDRV "A&ctualizare modúl-pilot…"
+    IDS_UPDATEDRV "A&ctualizează modulul-pilot…"
```

Comment:
The long infinitive conjugation for this verb „a actualiza” has been intentional (see the 4th point in [Romanian translation notes.txt](https://github.com/reactos/reactos/blob/master/media/doc/Romanian%20translation%20notes.txt)), as this is not a direct action, but merely a provider of an additional dialog box, and its marked as such by the ellipsis sign at the end. You'll notice that it's this way in the rest of codebase too. If you're ok with it, I'd suggest to revert this particular change, otherwise take note to also change the rest of translations in the future.